src/runtime_src/core/edge/user/zynq_dev.h

```diff
@@ -16,6 +16,8 @@
 #ifndef _XCL_ZYNQ_DEV_H_
 #define _XCL_ZYNQ_DEV_H_
 
+#include <boost/filesystem.hpp>
+#include <regex>
```

Comment:
These two should be added in the .cpp file where they are used, unless of course they are also used in the header file.